llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -859,10 +859,11 @@ static bool removeSYCLKernelsConstRefArray(GlobalVariable *GV) {
   for (auto It = IOperands.begin(); It != IOperands.end(); It++) {
```

Comment:
```suggestion
  for (auto *It = IOperands.begin(); It != IOperands.end(); It++) {
```

Reply:
The suggested change is NFC, both syntax variants mean the same here.
I prefer 'auto It = ...'. It is also way more typical syntax (about 2000 cases in sources vs 54 cases of 'auto *It = ' )